en/docs/security-announcements/cve-justifications/2025/CVE-2025-46392.md

```diff
@@ -0,0 +1,30 @@
+---
+title: CVE-2025-46392
```

Comment:
Let's map this doc to the following, 

- https://github.com/wso2/docs-security/blob/main/en/mkdocs.yml#L357
- https://github.com/wso2/docs-security/blob/main/en/docs/security-announcements/cve-justifications/2025/index.md?plain=1